examples/stress_tests/particles.rs

```diff
@@ -190,10 +193,21 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             .rollback_resource_with_clone::<ParticleRng>();
     }
 
-    app.checksum_component_with_hash::<Velocity>()
-        // todo: ideally we'd also be doing checksums for Transforms, but that's
-        // currently very clunky to do.
-        .insert_resource(args)
+    app.insert_resource(args)
+        // Components can be added to the frame checksum automatically if they implement Hash...
+        .checksum_component_with_hash::<Velocity>()
+        // ...or you can provide a custom hashing process
+        .checksum_component::<Transform>(|transform| {
+            let mut hasher = bevy::utils::FixedState.build_hasher();
+
+            // In this demo we only translate particles, so only that value
+            // needs to be tracked.
```

Comment:
Should we assert finite here, like we do for velocity?

Reply:
Agreed, I've also included a little panic message in the assertion explaining why it's there.